test/end2end_test.go

```diff
@@ -2671,6 +2671,45 @@ func testExceedMaxStreamsLimit(t *testing.T, e env) {
 	}
 }
 
+const defaultMaxStreamsClient = 100
+
+func TestClientExceedMaxStreamsLimit(t *testing.T) {
```

Comment:
How about `TestExceedDefaultMaxStreamsLimit`, as opposed to the previous test `TestExceedMaxStreamsLimit`.